src/example-types/esTwoLevelAggregation.js

```diff
@@ -31,17 +37,33 @@ module.exports = {
             )
           ),
           aggs: {
-            twoLevelAgg: {
-              [context.value_type]: _.omitBy(
-                _.isNil,
-                _.extend(
-                  {
-                    field: context.value_field,
-                  },
-                  context.value_data
+            ...(validMetrics
```

Comment:
We should unify the code paths here and treat `stats` as a metric agg (which it is)